src/VisualStudio/CSharp/Test/PersistentStorage/SQLiteV2PersistentStorageTests.cs

```diff
@@ -46,7 +46,7 @@ public async Task TestCrashInNewConnection()
 
             // Because instantiating the connection will fail, we will not get back
             // a working persistent storage. We are testing a fault recovery code path.
-            await using (var storage = await GetStorageAsync(solution, faultInjector, throwOnFailure: false))
+            await using (var storage = await GetStorageAsync(solution, faultInjector: faultInjector, throwOnFailure: false))
```

Comment:
Why?

Reply:
i added another optional parameter, and this positional one needed to become named.